deployments/mainnet/usdc/deploy.ts

```diff
@@ -11,14 +11,16 @@ export default async function deploy(deploymentManager: DeploymentManager, deplo
 
   // Deploy all Comet-related contracts
   const deployed = await deployComet(deploymentManager, deploySpec);
-  const { comet } = deployed;
+  // const { comet } = deployed;
 
   // Deploy Bulker
-  const bulker = await deploymentManager.deploy(
-    'bulker',
-    'Bulker.sol',
-    [await comet.governor(), comet.address, WETH.address]
-  );
+  // const bulker = await deploymentManager.deploy(
+  //   'bulker',
+  //   'Bulker.sol',
+  //   [await comet.governor(), comet.address, WETH.address]
+  // );
 
-  return { ...deployed, bulker };
+  // return { ...deployed, bulker };
```

Comment:
What's the plan for deploying bulker? Probably in a new branch/PR, we could comment out everything other than deploying bulker and do the deploy from there?

Reply:
We shouldn't need to comment the other lines

Reply:
Oh right, that's the magic of idempotent deploys 🪄